src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -144,15 +146,16 @@ def find_issues(self, path):
             lines = list(filter(None, output.decode('utf-8').split('\n')))
             payload = json.loads(lines[-1])
         except json.decoder.JSONDecodeError:
-            logger.warn('Invalid json output', path=path)
-            return
+            logger.warn('Invalid json output', path=path, lines=lines)
+            raise
 
-        full_path = os.path.join(self.repo_dir, path)
-        if full_path not in payload:
+        if full_path not in payload and path not in payload:
```

Comment:
The second condition here is redundant with the first, isn't it? Since `path` is a substring of `full_path`

Reply:
No `payload` is a dict, and you can get both keys: 

```json
{
  "/full/path/to/file": [],
  "file": [],
}
```

Reply:
Oh, right, I hadn't noticed. It's a pain that you can get both...